CONTRIBUTING.md

```diff
@@ -12,79 +12,107 @@ Moreover, React Cosmos tries to answer the question: **How to design components
 
 [The Best Code is No Code At All.](http://blog.codinghorror.com/the-best-code-is-no-code-at-all/) Start with this in mind before writing code.
 
-React Cosmos is a [monorepo](packages) with many small packages. This makes it possible to design isolated units, but also to build and test all packages together (inside one of the [examples](examples)).
+### Monorepo
+
+React Cosmos is a [monorepo](packages) powered by [Lerna](https://github.com/lerna/lerna). The monorepo structure makes it possible to publish independent packages while being able to have end-to-end tests for the whole project, and to easily link examples to unpublished packages.
+
+While the monorepo structure is powerful, it also creates limitations. Important to note:
+
+- **Test and build tools are installed globally in the root node_modules.** This includes Jest, Babel, webpack, and their corresponding plugins and loaders. ESLint is also applied globally.
+
+- **React and webpack deps are *hoisted* in the root node_modules.** The linked packages and examples are sibling directories, so they each have a separate node_modules dir. In real life, however, when Cosmos packages are installed in a user codebase, they all share a common node_modules. We simulate the real life scenario by deduplicate these dependencies.
```

Comment:
Noob question here -- I'm not at all familiar with Lerna, so I checked out the [docs for hoist](https://github.com/lerna/lerna/blob/master/doc/hoist.md) which describe it as a flag (`--hoist`). However,  I can't see that flag set anywhere in cosmos. Not sure why the flag would be needed anyway, since, by default, node looks in the root folder's node_modules for packages?